core/storage/suggestion/gae_models_test.py

```diff
@@ -2994,3 +2994,3029 @@ def test_get_model_association_to_user(self) -> None:
+
+    def test_apply_deletion_policy(self) -> None:
+        suggestion_models.QuestionReviewerTotalContributionStatsModel.create(
+            contributor_id=self.USER_ID_1,
+            topic_ids_with_question_reviews=(
+                self.TOPIC_IDS_WITH_QUESTION_REVIEWS),
+            reviewed_questions_count=self.REVIEWED_QUESTIONS_COUNT,
+            accepted_questions_count=self.ACCEPTED_QUESTIONS_COUNT,
+            accepted_questions_with_reviewer_edits_count=(
+                self.ACCEPTED_QUESTIONS_WITH_REVIEWER_EDITS_COUNT),
+            rejected_questions_count=self.REJECTED_QUESTIONS_COUNT,
+            first_contribution_date=self.FIRST_CONTRIBUTION_DATE,
+            last_contribution_date=self.LAST_CONTRIBUTION_DATE
+        )
+        self.assertTrue(
+            suggestion_models.QuestionReviewerTotalContributionStatsModel
+            .has_reference_to_user_id(self.USER_ID_1))
+
+        (
+            suggestion_models.QuestionReviewerTotalContributionStatsModel
+            .apply_deletion_policy(self.USER_ID_1)
+        )
+
+        self.assertFalse(
+            suggestion_models.QuestionReviewerTotalContributionStatsModel
+            .has_reference_to_user_id(self.USER_ID_1))
+
+    def test_export_data_trivial(self) -> None:
+        user_data = (
+            suggestion_models.QuestionReviewerTotalContributionStatsModel
+            .export_data('non_existent_user'))
+        self.assertEqual(user_data, {})
+
+    def test_export_data_nontrivial(self) -> None:
+        suggestion_models.QuestionReviewerTotalContributionStatsModel.create(
+            contributor_id=self.USER_ID_1,
+            topic_ids_with_question_reviews=(
+                self.TOPIC_IDS_WITH_QUESTION_REVIEWS),
+            reviewed_questions_count=self.REVIEWED_QUESTIONS_COUNT,
+            accepted_questions_count=self.ACCEPTED_QUESTIONS_COUNT,
+            accepted_questions_with_reviewer_edits_count=(
+                self.ACCEPTED_QUESTIONS_WITH_REVIEWER_EDITS_COUNT),
+            rejected_questions_count=self.REJECTED_QUESTIONS_COUNT,
+            first_contribution_date=self.FIRST_CONTRIBUTION_DATE,
+            last_contribution_date=self.LAST_CONTRIBUTION_DATE
+        )
+        expected_data = {
+            'topic_ids_with_question_reviews': (
+                self.TOPIC_IDS_WITH_QUESTION_REVIEWS),
+            'reviewed_questions_count': (
+                self.REVIEWED_QUESTIONS_COUNT),
+            'accepted_questions_count': (
+                self.ACCEPTED_QUESTIONS_COUNT),
+            'accepted_questions_with_reviewer_edits_count': (
+                self
+                .ACCEPTED_QUESTIONS_WITH_REVIEWER_EDITS_COUNT),
+            'rejected_questions_count': self.REJECTED_QUESTIONS_COUNT,
+            'first_contribution_date': (
+                self.FIRST_CONTRIBUTION_DATE.isoformat()),
+            'last_contribution_date': (
+                self.LAST_CONTRIBUTION_DATE.isoformat())
+            }
```

Comment:
Deindent this by 4 to match the opening {

Reply:
Okay.